mri/parallel_mri/extract_sensitivity_maps.py

```diff
@@ -94,25 +94,30 @@ def gridded_inverse_fourier_transform_nd(kspace_loc,
         pfft.ifftn(pfft.ifftshift(gridded_kspace))), 1, 0)
 
 
-def gridded_inverse_fourier_transform_stack(kspace_plane_loc, z_sample_loc,
-                                            kspace_data, grid, method):
+def gridded_inverse_fourier_transform_stack(kspace_data_sorted,
+                                            kspace_plane_loc,
+                                            idx_mask_z,
+                                            grid,
+                                            shape,
+                                            method):
     """
     This function calculates the gridded Inverse fourier transform
     from Interpolated non-Cartesian data into a cartesian grid. However,
-    the IFFT is done similar to Stacked FOurier transform.
-
+    the IFFT is done similar to Stacked Fourier transform.
     Parameters
     ----------
+    kspace_data_sorted: np.ndarray
+        The sorted k-space data corresponding to kspace_plane_loc above
     kspace_plane_loc: np.ndarray
         The N-D k_space locations of size [M, N]. These hold locations only
         in plane, extracted using get_stacks_fourier function
-    z_sample_loc: np.ndarray
-        This holds the z-sample locations for stacks. Again, extracted using
+    idx_mask_z: np.ndarray
+        contains the indices of the acquired Fourier plane. Extracted using
         get_stacks_fourier function
-    kspace_data: np.ndarray
-        The k-space data corresponding to kspace_plane_loc above
-    grid: np.ndarray
+    grid: tuple
```

Comment:
why did the type change?

Reply:
idx_mask_z is needed. z_sample_loc is redondant as we already have the kspace_data to give the z-locations.

Reply:
No I'm asking why the type of `grid` changed

Reply:
Because we realised it is a tuple (containing two arrays). 
Do you prefere if I precise tuple of np.array?

Reply:
ok the grid is expressed as a tuple of `ndarray`s?
Maybe not in the type, but in the description of the variable it would be nice.

Reply:
Ok!